test/configuration_test.dart

```diff
@@ -108,4 +108,58 @@ Future<void> main([List<String>? args]) async {
     expect(() => realm.write(() {}), throws<RealmException>("Can't perform transactions on read-only Realms."));
     realm.close();
   });
+
+  test('Configuration inMemory - no files after closing realm', () {
+    Configuration config = Configuration([Car.schema], inMemoryOnly: true);
+    var realm = Realm(config);
+    realm.write(() => realm.add(Car('Tesla')));
+    realm.close();
+    expect(Realm.existsSync(config.path), false);
+  });
+
+  test('Configuration inMemory - lost objects after closing', () {
```

Comment:
I think we could delete this test. Does not make much sense to test that if it is inMemory only.

Reply:
Deleted.